scripts/autograph_travis_test_config.yaml

```diff
@@ -1,254 +0,0 @@
-# Note: Most of the configuration here got copied from
-# https://github.com/mozilla-services/autograph/blob/master/autograph.yaml
-
```

Comment:
This file is used by `docker-compose.yml` for `autograph` on local envs as well, I don't think it should be removed (ideally it would be included by circle somehow but as you mentioned that doesn't seem to be possible). It should be renamed though.

Reply:
awww, sorry about that. The name was misleading and I probably grep'ed the wrong filename later.